api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -177,11 +177,9 @@ public static class AuthConfig {
 
   public static class WgsCohortExtractionConfig {
     public String serviceAccount;
-    public String serviceAccountTerraProxyGroup;
     public String operationalTerraWorkspaceNamespace;
     public String operationalTerraWorkspaceName;
     public String operationalTerraWorkspaceBucket;
-    public String extractionPetServiceAccount;
```

Comment:
Please do not remove flags that belong to workbench team. Please revert this change.

Reply:
done